site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -197,7 +213,7 @@ include::_tenant-generator-length-config-rules.adoc[]
 :generator_display_name!:
 +
 
-[field]#tenant.externalIdentifierConfiguration.registrationVerificationIdGenerator.type# [type]#[String]# [required]#Required# [since]#Available since 1.8.0#::
+[field]#tenant.externalIdentifierConfiguration.registrationVerificationIdGenerator.type# [type]#[String]# [required]#Required# [default]#randomAlphaNumeric# [since]#Available since 1.8.0#::
```

Comment:
Use back ticks and it should be "Defaults to <value>" - see other usages. 

If it was previously required, the default will be new as of 1.28.0. So you could add a note in description. 

Such as 

> The description of the field...
> 
> Prior to version `1.28.0` this value was required.